gui/package.json

```diff
@@ -8,8 +8,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "engines": {
-    "node": ">=14.0.0",
-    "npm": ">=6.0.0"
+    "node": ">=16.0.0",
```

Comment:
Not sure about this. Maybe we can't use it anymore on raspberry / Debian

Reply:
Not sure, it was merged from main from recent updates.

Reply:
Let's keep it as it is and in case of problems we can have another look. I prefer up to date software